src/main/java/io/personium/core/model/DavCommon.java

```diff
@@ -41,25 +41,29 @@ private DavCommon() {
     /** Overwrite Value when header overwrite is not allowed.*/
     public static final String OVERWRITE_FALSE = "F";
 
+    public static Pattern PATTERN_CTRL_CHARS = Pattern.compile("\\p{C}");
```

Comment:
I cannot find reference about `\\p{C}`. 

Could you show me some reference?

Reply:
How about this? 

```suggestion
    public static Pattern PATTERN_CTRL_CHARS = Pattern.compile("\\p{Cntrl}");
```

Reply:
ping @shimono

Reply:
https://docs.oracle.com/javase/jp/17/docs/api/java.base/java/util/regex/Pattern.html